src/dbus/server/dbus_thread_object_rcp.cpp

```diff
@@ -2010,7 +2010,16 @@ void DBusThreadObjectRcp::DeactivateEphemeralKeyModeHandler(DBusRequest &aReques
     otError error        = OT_ERROR_NONE;
     auto    threadHelper = mHost.GetThreadHelper();
 
+    VerifyOrExit(!mBorderAgent.GetEphemeralKeyEnabled(), error = OT_ERROR_INVALID_STATE);
+
+    otbrLogInfo("Deactivating Ephemeral Key. Border Agent State: %s",
+                otBorderAgentGetState(threadHelper->GetInstance()) ? "active" : "stopped");
```

Comment:
> otBorderAgentGetState(threadHelper->GetInstance()) ? "active" : "stopped"

- It's error prone to leverage the priori that the `OT_BORDER_AGENT_STATE_STOPPED` equals to 0.
- There are 3 states in `otBorderAgentState` instead of two. Maybe you need a helper function to convert it to a string.

Reply:
Separated `stop` and `active` with different error messages. So we could have the debug info via host without logs. Thanks~